pkg/config/agent/config.go

```diff
@@ -404,3 +406,24 @@ type OVSBridgeConfig struct {
 	// only a single physical interface is supported.
 	PhysicalInterfaces []string `yaml:"physicalInterfaces,omitempty"`
 }
+
+type PingMonitoringToolConfig struct {
```

Comment:
As we discussed, enabling & configuring "Ping Monitoring" should be done through an API (e.g. single instance CRD). This will enable easier consumption, and will facilitate enabling the tool on the fly in a live cluster when it is needed / desired.

Reply:
Yes this was clarified during yesterday's offline sync with @IRONICBo

Reply:
Now new commit support to start/stop `NodeLatencyMonitor` with singleton CRD.